package.json

```diff
@@ -277,7 +277,7 @@
     "cookie": "^0.4.1",
     "core-js": "^3.18.0",
     "date-fns": "^2.24.0",
-    "date-fns-tz": "^1.1.1",
+    "date-fns-tz": "^2.0.0",
```

Comment:
this upgrade was required to use their `formatInTimeZone` function. i also figured this may be necessary if we're migrating from `moment` to `date-fns` Platform-wide